newsApi.test.js

```diff
@@ -0,0 +1,17 @@
+const NewsApi = require('./newsApi');
+
+require("jest-fetch-mock").enableMocks();
+
+describe('NewsApi', () => {
+  it('uses fetch to obtain articles', async () => {
+    const api = new NewsApi();
+
+    fetch.mockResponseOnce(
+      JSON.stringify(["This is a test article"])
```

Comment:
Perhaps pass it a JS object to make it more realistic. Then you'll be able to test the logic a bit more by checking values against key names